src/flutter/shell/platform/linux_embedded/surface/native_window_drm.cc

```diff
@@ -27,6 +27,12 @@ NativeWindowDrm::NativeWindowDrm(const char* deviceFilename) {
     LINUXES_LOG(ERROR) << "Couldn't open " << deviceFilename;
     return;
   }
+
+  if (!drmIsMaster(drm_device_)) {
```

Comment:
Did you test on Raspberry Pi4? Are there any version restrictions?

Reply:
I tested on Raspberry Pi4 and it worked fine on the DRM backend.
[According to the release information](https://lists.x.org/archives/xorg-announce/2019-April/002992.html), `drmIsMaster()` was added in libdrm 2.4.98.